package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@datadog/native-iast-taint-tracking",
   "home": "https://github.com/DataDog/dd-native-iast-taint-tracking-js/blob/main/README.md",
   "repository": "git@github.com:DataDog/dd-native-iast-taint-tracking-js.git",
-  "version": "1.1.1",
+  "version": "1.1.2",
```

Comment:
Since we added a new function to the API Wouldn´t it be better to change the minor? 1.2.0?